### ISSUE TITLE
Bug/Breakpoints container - color on example

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#701]
On some of the examples, we are showing a light blue background with white text.  This does not meet the contrast ratio we need for accessibility.

![image](https://user-images.githubusercontent.com/24397282/168888302-421629cd-1325-49c1-8efc-33ca239b2ea9.png)

Solution: Everywhere we use this color as a background on the page, make the text primary-700 (#1A1E51).

![image](https://user-images.githubusercontent.com/24397282/168888754-24e90df9-9cfe-4e5d-afda-acdf7fef8a2b.png)
